tests/integration/test_printenv.sh

```diff
@@ -32,7 +32,7 @@ trap teardown EXIT
 
 OUTPUT=$(/bin/true)
 
-${ATOMIC} run atomic-test-5 | grep -v ^NAME= | grep -v ^IMAGE= | grep -v printenv | grep -v atomic | grep -v coverage | sort > ${TESTDIR}/atomic-test-5.1
+${ATOMIC} run atomic-test-5 | grep -v ^NAME= | grep -v ^IMAGE= | grep -v ^SUDO | grep -v printenv | grep -v atomic | grep -v coverage | sort > ${TESTDIR}/atomic-test-5.1
 
 printenv | grep -v printenv | grep -v atomic | grep -v coverage | sort > ${TESTDIR}/atomic-test-5.2
```

Comment:
In the previous comparison, you grep "out" SUDO vars.  Should you do the same here as well?